tests/fast-integration/compass-test/compass-test.js

```diff
@@ -42,7 +42,7 @@ describe("Kyma with Compass test", async function() {
       return;
     }
 
-    await installer.installKyma({skipComponents: componentsToSkip, withCompass: true, withCentralAppConnectivity: withCentralAppConnectivity});
+    await installer.installKyma({withCompass: true, withCentralAppConnectivity: withCentralAppConnectivity, componentsToSkip});
```

Comment:
The shorthand for object initialization works when key and value match (you can skip value). In case of `skipComponents` it is not a case. Rename `componentsToSkip` variable or keep the long syntax.